CONTRIBUTING.md

```diff
@@ -4,9 +4,27 @@
 
 <em lang="de">Die <a href="#deutsch">deutsche Übersetzung</a> findest du weiter unten.</em>
 
-## English
-
-### [Code of conduct](https://www.ecamp3.ch/en/code-of-conduct)
+# English
+
+## [Code of conduct](https://www.ecamp3.ch/en/code-of-conduct)
+
+## Workflow
+This is a basic Overview of the Workflow, more Information about how to set up a development environment on your Computer is in the [Wiki](https://github.com/ecamp/ecamp3/wiki/installation).
+If something about the setup is unclear or you run into an error, there are [discussions](https://github.com/ecamp/ecamp3/discussions) on GitHub for you to ask questions and ask for help.
+### Labels
+Issues are Marked with Labels and some of them are not self-explanatory and explained here:
+- **Type-Labels**:
+Type labels follow the `type: *` format with the options `Frontend`, `Print`, `Deployment` & `API` the architecture for those are partially Documented in the [Wiki](https://github.com/ecamp/ecamp3/wiki/architecture-frontend)
+- **Needs prototype**: This Issue needs a Prototype first, don't start fully implementing this Feature yet. A Prototype can be many things, if your Prototype is an Sketch, Mockup, Partial Implementation or something else is up to you.
```

Comment:
Is the responsibility to create a prototype with the contributor, or do contributors have to wait for core team members to provide a prototype?

Reply:
This is for features that are generally something the core team wants implemented but there are some vague requirements like in #3886. To ensure everyone is on the same page about what an implementation could/should look like there should be a prototype of some sort. I think it is good if the implementing dev is the one to provide the prototype but that's an opinion and not a requirement. One could skip this step at the cost of being confronted with change requests about their implementations of the interpretation of specifications in addition to the usual PR-Discussions. Thats my interpretation of the label, that was originally created by @manuelmeister afaik so correct me if i am mistaken. My answer to your two questions is no, they dont have to be the one to create the prototype but they dont have to wait on core team to come up with a prototype. I modified the phrasing a bit, lmk what you think.